edb/server/protocol/ai_ext.py

```diff
@@ -1591,19 +1949,24 @@ async def _handle_rag_request(
             if custom_prompt:
                 if not isinstance(custom_prompt, list):
                     raise TypeError(
-                        "prompt.custom must be a list of {role, content} "
-                        "objects"
+                        "prompt.custom must be a list, where each element is \
+                            one of the following types: \
+                        - EdgeDBSystemMessage: { role, content } \
```

Comment:
Where does `EdgeDBSystemMessage` and others come from?  Seems a bit confusing since those aren't actual keys in input?

Reply:
Yeah, but this is server-side.  If one interacts with the API via another SDK the errors would not make sense.

Reply:
I get it, I believe the error is better now. It should be understandable enough for everyone.